test/Unity.TypedFactories.Tests/UnityTypedFactoryTests.cs

```diff
@@ -184,7 +186,7 @@ public void given_instantiated_Sut_when_Create_is_called_with_Type_parameter_the
         }
 
         [Test]
-        [ExpectedException(typeof(ResolutionFailedException))]
+        //[ExpectedException(typeof(ResolutionFailedException))]
```

Comment:
This is misleading since the expected exception seems to be `InvalidOperationException`

Reply:
I made the requested changes.